src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Metadata/ModelBindingMessageProvider.cs

```diff
@@ -91,6 +94,24 @@ public Func<string> MissingKeyOrValueAccessor
         }
 
         /// <inheritdoc/>
+        public Func<string> MissingRequestBodyRequiredValueAccessor
```

Comment:
Look at existing tests that reference properties in this class. We test each to confirm changing them actually changes errors in the expected cases. In this case, it would be a check in the other `InputFormatterTest` class.